recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "iris-esmf-regrid" %}
-{% set version = "0.9.0" %}
+{% set version = "0.10.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/SciTools-incubator/iris-esmf-regrid/archive/v{{ version }}.tar.gz
-  sha256: b8b479ae93ed77738592258f54bd3a2cc9aff1804ba712e2ad2a133563d0c378
+  sha256: 1c696bb54c5703a938bde6368bbc0301d93659527b32b5f7200dcff824cf1743
```

Comment:
Are you sure about this hash?

[When I look on Pip](https://pypi.org/project/esmf-regrid/#copy-hash-modal-ffff3c27-21cc-4586-a874-01f309b6b634) I get: `37e85a54f7951ec81c7b2fd55be5907793e5894870f78a9e1411fd0b9ed64bfe`

Reply:
Oh woops I see this is actually sourcing directly from GitHub. Could we switch to sourcing from Pip in line with latest SciTools practice elsewhere?

Reply:
I got this sha from the github release, which is where the url is pointing